package.json

```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "build": "tsdx build",
-    "test": "tsdx test",
+    "test": "tsdx test --runInBand",
```

Comment:
Slows things down a bit but noticed funkyness when the app and multichainApp integration tests were running at the same time